### PR TITLE
kubeturbo operator support for http proxy

### PR DESCRIPTION
**Pull request by esara**
```
helm install kubeturbo kubeturbo --dry-run --debug --set serverMeta.proxy=http://proxy:8080
---
# Source: kubeturbo/templates/configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: turbo-config-kubeturbo
data:
  turbo.config: |-
    {
      "communicationConfig": {
        "serverMeta": {
          "version": "7.21",
          "turboServer": "https://Turbo_server_URL"
          "proxy": "http://proxy:8080"
        },
        "restAPIConfig": {
          "opsManagerUserName": "Turbo_username",
          "opsManagerPassword": "Turbo_password"
        }
      },
      "HANodeConfig": {
        "nodeRoles": ["master"]
      }
    }
...
```

### CODE REVIEW

File: deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/configmap.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ data:
         "serverMeta": {
           "version": "{{ .Values.serverMeta.version }}",
           "turboServer": "{{ .Values.serverMeta.turboServer }}"
+        {{- if .Values.serverMeta.proxy }}
```
Comment (enlinxu) [inline deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/configmap.yaml]: Missing a comma after turboServer if proxy exists.

File: deploy/kubeturbo/templates/configmap.yaml [hunk 2]
```diff
@@ -9,6 +9,9 @@ data:
         "serverMeta": {
           "version": "{{ .Values.serverMeta.version }}",
           "turboServer": "{{ .Values.serverMeta.turboServer }}"
+        {{- if .Values.serverMeta.proxy }}
```
Comment (enlinxu) [inline deploy/kubeturbo/templates/configmap.yaml]: Same as above

File: deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/configmap.yaml (hunk 1 above)
Comment (esara) [reply deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/configmap.yaml]: thanks for catching it - pushing an update

File: deploy/kubeturbo/templates/configmap.yaml (hunk 2 above)
Comment (esara) [reply deploy/kubeturbo/templates/configmap.yaml]: same

**Pull request merged**